_boskos.yaml

```diff
@@ -0,0 +1,860 @@
+# generated with generate-boskos.py; do not edit directly
```

Comment:
Pretty sure this file should not be here.

Reply:
I have edited the `generate-boskos.py` file to add the `azuremag-quota-slice` which has generated the `_boskos.yaml` file.

Reply:
That script generates `core-services/prow/02_config/_boskos.yaml`.